recipes-framework/onnxruntime/onnxruntime_1.18.0.bb

```diff
@@ -60,92 +59,93 @@ OECMAKE_CXX_FLAGS += "-I${PYTHON_INCLUDE_DIR} -I${PYBIND11_INCLUDE} -I${NUMPY_IN
 OECMAKE_CXX_FLAGS_RELEASE += "-I${PYTHON_INCLUDE_DIR} -I${PYBIND11_INCLUDE} -I${NUMPY_INCLUDE}"
 
 EXTRA_OECMAKE:append = " \
-    -Donnxruntime_RUN_ONNX_TESTS=OFF \
-    -Donnxruntime_GENERATE_TEST_REPORTS=ON \
-    -Donnxruntime_USE_MIMALLOC=OFF \
-    -Donnxruntime_ENABLE_PYTHON=ON \
-    -DPython_EXECUTABLE=${STAGING_BINDIR_NATIVE}/${PYTHON_PN}-native/${PYTHON_PN} \
-    -DPYTHON_EXECUTABLE=${STAGING_BINDIR_NATIVE}/${PYTHON_PN}-native/${PYTHON_PN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_FLAGS=-Wno-error=array-bounds \
+    -DCMAKE_CXX_FLAGS=-Wno-error=maybe-uninitialize \
```

Comment:
Sorry. There seems to have been a mistake in the original flag.
The correct flag is `maybe-uninitialized`, not `maybe-uninitialize`.
https://github.com/NobuoTsukamoto/meta-onnxruntime/actions/runs/9534931175/job/26280013061?pr=6#step:5:5642